zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -867,6 +896,20 @@ function NotebookCtrl ($scope, $route, $routeParams, $location, $rootScope,
     angular.element('.ace_autocomplete').hide()
   })
 
+  $scope.isOwnerEmpty = function() {
+    if ($scope.permissions.owners.length > 0) {
+      for (let i = 0; i < $scope.permissions.owners.length; i++) {
+        $scope.permissions.owners[i] = $scope.permissions.owners[i].trim()
```

Comment:
@felixcheung Thanks for review and useful comments!
You're right! I initially think something wrong. Let me modify this part again!
Thanks a lots!